### DISCUSSION TITLE
What was America's first national park?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#59] [Galaxy Trivia] [answered]
**Hint:** ♨️

**Comment by Tynab** [comment, marked as answer]
Yellowstone National Park was America's first national park. It was established by an act of Congress and signed into law by President Ulysses S. Grant on March 1, 1872. Yellowstone is located primarily in the U.S. state of Wyoming, although it also extends into Montana and Idaho. It is known for its geothermal features, including the famous Old Faithful geyser, as well as its wildlife and natural beauty. Today, the United States has 63 national parks, as well as numerous other protected areas and wilderness reserves.
